File: modulo1/segmento3/apunte3/Ejercicio3.java
import java.util.Scanner;

public class Ejercicio3{
	public static void main(String args[]){
		Scanner teclado = new Scanner(System.in);
	
		String opcion;

		System.out.println("Ingrese un caracter cualquiera : ");
		String caracter = teclado.nextLine();


		if(caracter.toUpperCase().equals("A") || caracter.toUpperCase().equals("E") || caracter.toUpperCase().equals("I") || caracter.toUpperCase().equals("O") || caracter.toUpperCase().equals("U")){
			opcion = "vocal";
		}else{
			opcion="consonante";
		}
		System.out.println(caracter.toString() + " es " + opcion );

	}	

}

File: modulo1/segmento3/apunte3IfElse/ejercicio3.py
car = input("ingrese un caracter cualquiera: ")

if (car == "a" or car == "e" or car == "i" or car == "o" or car == "u" ):
    opcion = "vocal"
elif (car.isdigit()):
    opcion = "numero"
else :
    opcion = "consonante"

print (opcion)


